src/main/haskell/kore/src/Kore/Step/BaseStep.hs

```diff
@@ -237,30 +244,38 @@ stepWithAxiom
         , substitution
         , _  -- TODO: Use this proof
         ) <-
-            normalizeUnificationError existingVars
+            normalizeUnificationError
+                (makeFalsePredicate, [], EmptyUnificationProof)
```

Comment:
Test this?

Reply:
It will be tested once this is solved: https://trello.com/c/dTi0lICM